internal/lib/lib.go

```diff
@@ -46,6 +46,12 @@ var shardSizeMap = map[string]uint32{
 	"SMALL":  1,
 }
 
+var fqdnEnum = map[string]int32{
```

Comment:
Shall we use uint32 instead?

Reply:
any benefit?

Reply:
Just that the value can never be negative, so thought uint32 would be better.

Reply:
It's an internal constant and we can directly do comparisons for this number, if it's uint32 - we will have to cast and compare for example if lib.GetL4FqdnFormat() == uint32(3)